README.markdown

```diff
@@ -617,6 +636,26 @@ if let textContainer = self.textContainer {
 }
 ```
 
+While it is fine to split an `if let` into multiple lines when several variables are present, the first variable should be delcared on the same line as the if:
+
+**Preferred:**
+```swift
+if let subview = subview,
+   volume = volume {
```

Comment:
Using this pattern makes you fight Xcode ever so slightly. After typing the comma and new line on the first line, Xcode will start the cursor under the "i" in `if`. If you type a tab, you will get shifted four spaces over, to under the "e" in `let`, rather than under the "l".


![tabs](https://cloud.githubusercontent.com/assets/12215/20074804/c133340e-a4ff-11e6-9cb0-8b5903d15312.gif)